File: server/src/main/java/ru/onlinewallet/entity/user/UserSettings.java
package ru.onlinewallet.entity.user;

import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@Entity
@NoArgsConstructor
@Table(name = "w_user_settings")
@PersistenceUnit(unitName = "postgres")
public class UserSettings {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "user_id")
    private Long userId;

    @Column(name = "first_name")
    private String firstName;

    @Column(name = "last_name")
    private String lastName;

    @Column(name = "middle_name")
    private String middleName;

    @Column
    private String about;

    @Column
    private String url;

    @Column
    private String phone;

    @Column
    private String country;

    @Column
    private String language;

    @Column
    private String currency;

}

File: frontend/src/app/entities/user-settings.ts
export class UserSettings {
  id: number;
  userId: number;
  firstName: string;
  lastName: string;
  middleName: string;
  about: string;
  url: string;
  phone: string;
  country: string;
  language: string;
  currency: string;
  profileImage: any;

  constructor(json: any) {
    this.id = json.id;
    this.userId = json.userId;
    this.firstName = json.firstName;
    this.lastName = json.lastName;
    this.middleName = json.middleName;
    this.about = json.about;
    this.url = json.url;
    this.phone = json.phone;
    this.country = json.country;
    this.language = json.language;
    this.currency = json.currency;
    this.profileImage=json.profileImage
    console.log(json)
  }
}
